src/windowsservercore/ltsc2019/webassembly/Dockerfile

```
@@ -33,4 +59,31 @@ RUN msiexec /i %TEMP%\nodejs.msi /quiet /passive /qn /norestart
 # install jsvu and engines
 RUN npm install jsvu -g
 RUN npm exec -c "jsvu --os=win64 --engines=v8,spidermonkey"
-RUN setx PATH "%PATH%;%USERPROFILE%\.jsvu"
+RUN setx /M PATH "%PATH%;%USERPROFILE%\.jsvu"
+
+# Install cmake
+ENV CMAKE_VERSION 3.20.3
+RUN curl -SL --output %TEMP%\cmake-win.zip https://github.com/Kitware/CMake/releases/download/v%CMAKE_VERSION%/cmake-%CMAKE_VERSION%-windows-x86_64.zip \
+    && mkdir C:\cmake \
+    && tar -C C:\cmake -zxf %TEMP%\cmake-win.zip \
+    && setx /M PATH "%PATH%;C:\cmake\cmake-%CMAKE_VERSION%-windows-x86_64\bin" \
+    && del /q %TEMP%\cmake-win.zip
+
+# Install Visual Studio .NET and C++ tools
+RUN curl -SL --output vs_buildtools.exe https://aka.ms/vs/15/release/vs_buildtools.exe \
```

Pretty sure this is why the install isn't working for you.

```suggestion
RUN curl -SL --output vs_buildtools.exe https://aka.ms/vs/16/release/vs_buildtools.exe \
```

---

Great catch, thanks!